import/popups/PopupOpener.qml

```diff
@@ -366,6 +412,7 @@ Timer {
 
         var obj = null
         if (isDialog) {
+            console.log("openPopup: dialog");
```

Comment:
debugging logs should be removed I think, a couple more below also

Reply:
Thanks for spotting these. I've removed the ones I could see.